common/utils.js

```diff
@@ -529,6 +529,31 @@ const parseTrustedChains = (config, trustedChains) => {
     return trustedChains.length === 1 && trustedChains[0] === 'all' ? itsEdgeChains(config) : trustedChains;
 };
 
+function stellarAddressToBytes(address) {
```

Comment:
```suggestion
function asciiToBytes(string) {
```
make it more generic wrt what it's actually doing. Other chains might do the same too

Reply:
make sense, updated.